Add Azure lite plan (#37)
kyma-project/control-plane: docs/kyma-environment-broker/03-06-runtime-overrides.md

    data:
      database.password: YWRtaW4xMjMK
    ```  

### Disable overrides for specific plans

> Config Maps and Secrets overrides for customization Kyma Runtime works for all plans but for some kinds of special plans like for example "AzureLite"

Reviewer: ```suggestion
ConfigMaps and Secrets overrides work for all plans, however, you can disable overrides for specific lite plans, such as `AzureLite`.
```

> overrides can be disabled. To disable a specific override for a special lite plan use label `default-for-lite: "true"`.

Reviewer: ```suggestion
To disable a specific override for a lite plan, use the `default-for-lite: "true"` label.
```
